README.md

```diff
@@ -22,12 +22,18 @@ To build the docs, on Linux/ Mac OS X:
 make html
 ```
 
-On Windows:
+Or on Windows:
 
+Using Command Prompt:
```

Comment:
Good idea to include this! Can I suggest formatting it slightly differently to make it clear that both commands are for Windows:
```suggestion
* Using Command Prompt:
```